packages/devtools_app/lib/src/shared/eval_on_dart_library.dart

```diff
@@ -332,15 +319,15 @@ class EvalOnDartLibrary extends DisposableController
 
   static int _nextAsyncEvalId = 0;
 
-  EvalOnDartLibrary _dartDeveloperEvalCache;
+  EvalOnDartLibrary? _dartDeveloperEvalCache;
   EvalOnDartLibrary get _dartDeveloperEval {
     return _dartDeveloperEvalCache ??= EvalOnDartLibrary(
       'dart:developer',
       service,
     );
   }
 
-  EvalOnDartLibrary _widgetInspectorEvalCache;
+  EvalOnDartLibrary? _widgetInspectorEvalCache;
```

Comment:
can we make these late instead of non-nullable? if the references to these private vars in the dispose method are the issue, we may be able to get around that by having the dispose method reference the getters instead of the private vars. If the `??=` isn't allowed on a late variable, then we'll probably need to leave as is.

Reply:
I would make it lazy late if not disposing. 
It is not possible to detect if a late variable is initialized. Current implementation is the best possible.
https://stackoverflow.com/questions/67197902/how-to-check-late-variable-is-initialized-in-dart